share/man/man4/puc.4

```diff
@@ -45,6 +45,35 @@ driver.
 The list of supported devices is in
 .Pa sys/dev/puc/pucdata.c .
 Support for new cards should be added there.
+.Sh HARDWARE
+The
+.Nm
+driver supports the following PCI serial and parallel ports:
+.Pp
+.Bl -tag -width Ds -offset indent -compact
```

Comment:
```suggestion
.Bl -bullet -compact
```
You'll want to use a bullet list so it makes a subsection, this is one of the low hanging fruits of the hardware release notes.

Bullet lists require all the `.It`s on their own line.

You'll want to not use offset since some entries in this list is pretty wide already. MANWIDTH=80 is the reference standard, and we try to consider 59.